provider_config_test.go

```diff
@@ -12,7 +13,7 @@ type testProviderConfig struct {
 	throwError  bool
 }
 
-func (t *testProviderConfig) Initialize(jc *jwtConfig) error {
+func (t *testProviderConfig) Initialize(_ context.Context, jc *jwtConfig) error {
```

Comment:
Since we're passing a  `context.Background()` into `NewProviderConfig` in the tests already, I think that we can use the `ctx` provided to this func into the calls below (instead of ignoring the param and init'ing new contexts within the func).

Reply:
Whoops, totally misread that!